frontend/src/components/SortableTaskList.module.scss

```diff
@@ -34,3 +34,20 @@
   position: sticky;
   top: 0;
 }
+
+.taskName {
+  width: 210px;
+  max-height: 40px;
+  text-align: left;
+  overflow: hidden;
+  word-wrap: break-word;
+
+  -webkit-box-orient: vertical;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+}
+
+.measureTaskName {
+  @extend .taskName;
+  visibility: hidden;
```

Comment:
This seems to have less width available (about 163px)

This leads to the gaps closing sometimes when the name length falls within that difference